FogCreek/Glitch-Community
# Nova chance: Fix issues that were causing ssr not to apply

## Links
- https://nova-chance.glitch.me/
- https://app.zenhub.com/workspaces/community-5d07d1359dee6059a9688176/issues/fogcreek/glitch-community-work/228
- https://app.zenhub.com/workspaces/community-5d07d1359dee6059a9688176/issues/fogcreek/glitch-community-work/216

## Changes:
- Added useWindowSize which monitors the window innerWidth and innerHeight
- Reset the useUniqueId counter and call react-tabs.resetIdCounter before server rendering
- Expanded a class name because the server and client treat single digit number classes differently
- Treat /index.html like / for ssr and wistia preload

## How To Test:
Try loading /, /create, and /index.html. They should all have wistia tags in the body, html content in the network response, and there shouldn't be any react warnings about mismatches
